modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/strapon.dm

```diff
@@ -154,25 +154,25 @@
 					W.strapon_type = strapon_type
 					W.update_icon_state()
 					W.update_icon()
-					C.visible_message("<span class='notice'>[user] takes a strapon in their hand. They look menacingly</span>")
+					C.visible_message("<span class='notice'>[user] holds the strapon in their hand menacingly.</span>")
 					in_hands = TRUE
 					return
 		else
-			C.visible_message("<span class='notice'>[user]'s hand not empty. Can't take strapon in hand</span>")
+			C.visible_message("<span class='notice'>[user] tries to hold the strapon in their hand, but their hand isn't empty!</span>")
```

Comment:
`[user] tries to hold the strapon in their hand, but they seem quite occupied already.`